Source/WebCore/platform/animation/AcceleratedEffectValues.cpp

```diff
@@ -40,29 +40,22 @@
 namespace WebCore {
 
 AcceleratedEffectValues::AcceleratedEffectValues(const AcceleratedEffectValues& src)
```

Comment:
Is there any trick we can do to get this to be generated. Seems annoying to have to write it out just for those `copyRef` calls.